src/libMessage/Messenger.h

```diff
@@ -409,6 +409,41 @@ class Messenger
         const uint32_t consensusID, const std::vector<unsigned char>& blockHash,
         uint16_t& backupID, std::vector<unsigned char>& errorMsg,
         const std::deque<std::pair<PubKey, Peer>>& committeeKeys);
+
+    static bool SetLookupGetMicroBlockFromLookup(
+        std::vector<unsigned char>& dest, unsigned int offset,
```

Comment:
Sorry to be fussy, but to be consistent with the other functions, please rename `dest` to `dst`, and make offset `const`, and make the other arguments `const` too.